app/utilities/schema.py

```diff
@@ -196,8 +223,12 @@ def _load_schema_file(schema_name: str, language_code: str) -> Any:
 
 @lru_cache(maxsize=None)
 def load_schema_from_url(
-    schema_url: str, language_code: str | None
+    *, schema_url: str, language_code: str | None
```

Comment:
Similar to the previous PRs comment, happy for this to be as is, but could be enforced for language code only given function name is clear.

Reply:
changed 👍 will leave existing calls as is though